docs/legal.md

```diff
@@ -18,7 +18,6 @@ The logo is a composition of the [United Nations Accessibility Logo](https://www
 
 In its UMD bundle ally.js contains resources of the following projects:
 
-* [array.prototype.findindex](https://github.com/paulmillr/Array.prototype.findIndex), also MIT license
```

Comment:
this line should be maintained and ``, removed in `v#master` `` should be appended